iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractBus.java

```diff
@@ -52,8 +52,10 @@ public VoltageLevel getVoltageLevel() {
         return voltageLevel;
     }
 
+    @Override
```

Comment:
Not necessary as it is already added to ConfiguredBusImpl and CalculaedBusImpl

Reply:
It is already added in ConfiguredBusImpl and CalculatedBusImpl but the methods using it in AbstractBus assume it returns a TerminalExt and not a Terminal. As mentionned earlier, these methods can deal with a Terminal though.